cranelift/filetests/src/zkasm_runner.rs

```diff
@@ -74,15 +74,24 @@ pub fn run_zkasm_path(input_path: &Path) -> anyhow::Result<Vec<ExecutionResult>>
     )?;
 
     let mut output_file = NamedTempFile::new()?;
+    let common_args = [
+        "--prefix",
+        "../../tests/zkasm",
+        "test",
+        input_path.to_str().unwrap(),
+        output_file.path().to_str().unwrap(),
+    ];
+
+    #[cfg(target_os = "windows")]
+    let output = std::process::Command::new("cmd")
+        .args(["/C", "npm"])
```

Comment:
> This PR customized the path to NPM that we use on the Windows platform.

Isn't it the same path, but another way of invoking `npm`? That's my interpretation based on [this post](https://stackoverflow.com/a/515380), though I'm not familiar with running commands on Windows.

Reply:
I'm also confused by this - my understanding so far, is that direct `npm` invocation does not work because of the [different handling of PATH variable on Windows](https://github.com/rust-lang/rust/issues/37519).
Adding `cmd` here is supposed to add another layer of indirection that allows the GitHub action runner to inject the correct PATH variable (which is also defined in `GITHUB_PATH` workflow variable).

I struggle to find any decent documentation about this PATH behaviour, but FWIW, the example from Rust docs uses `cmd` on Windows: https://doc.rust-lang.org/nightly/std/process/struct.Command.html

Reply:
I don't necessarily disagree with the fix, but beware that invoking through `cmd` like this can change the interpretation of the arguments. This can lead to some extremely frustrating to debug problems later down the line.

If it does work currently, then its fine, of course, especially since nobody out of us seriously uses windows for development and thus GHA is the only thing that needs to work. Using `which` as suggested in the referenced PR sounds like it may be a straightforward improvement in the reliability aspect (though with its own tradeoffs -- adding a dependency potentially means more difficult merges down the line.)

> Adding cmd here is supposed to add another layer of indirection that allows the GitHub action runner to inject the correct PATH variable (which is also defined in GITHUB_PATH workflow variable).

FWIW the rust test runner is already invoked by `cmd` so the `%PATH%` should be already correct.